elife.cfg

```diff
@@ -1,6 +1,7 @@
 [general]
 env: dev
+#upload_path uploads
+#cache_path: cache
 
 [lax]
 location: /srv/lax/
-cache_path: /home/elife
```

Comment:
I'd prefer to keep ci/deployment details out of the application, I don't even like environment details being present, so what if I updated project_tests.sh to emit a better `app.cfg` file?